.changeset/soft-pens-arrive.md

```diff
@@ -0,0 +1,9 @@
+---
+"create-fuels": minor
+---
+
+feat: add wallet connections via the wallet SDK to the create-fuels template
+
+Users can now connect browser wallets like the Fuel Wallet and Fuelet to the create-fuels template. Under the hood, the template uses the [Fuel Wallet SDK's React Hooks](https://wallet.fuel.network/docs/dev/hooks-reference/).
+
+Having a browser wallet installed is not required to use the template, as it falls back to a local burner wallet if no browser wallet is connected.
```

Comment:
Given that we'll probably be merging #1872 in this release, it'd be good to already have the changesets conform to its assumptions, namely that the changeset description should only contain the PR title in it (`feat: add browser wallet connections to the create-fuels template app`).